src/controller_examples/rabbitmq_controller/proof/common.rs

```diff
@@ -17,10 +18,10 @@ use vstd::prelude::*;
 
 verus! {
 
-pub type ClusterState = State<RabbitmqClusterView, RabbitmqReconcileState>;
+pub type ClusterState = State<RabbitmqClusterView, RabbitmqReconciler>;
```

Comment:
Is it true that all the changes inside rabbitmq_controller::proof are this type of refactoring?

Reply:
What do you mean?

Reply:
I mean, I see you changed some type name and function name. Are all of these changes in the proof for such refactoring? I just want to see how careful I should be when going through these changes.

Reply:
They are not all of the same type. And if the refactoring is wrong, the compiler will complain.